.storybook/preview-head.html

```diff
@@ -4,4 +4,5 @@
 <link href="https://www.qld.gov.au/__data/assets/css_file_folder/0015/110823/qg-components.css?v=0.1.10" rel="stylesheet">
 <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
 <link rel="stylesheet" href="https://unpkg.com/formiojs@latest/dist/formio.full.min.css">
+<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```

Comment:
The latest version is 3.6.0. Is there something blocking us from using it?

Reply:
carl's right, we are on 3.6.0 in swe
https://github.com/qld-gov-au/qg-web-template/blob/master/package.json#L113

Reply:
i didn't think about the jQuery version, i copied this veriosn 3.3.1 from the scrpt which included in squiz matrix page, as i tried to simulate the squiz matrix environment
eg. view-source:https://www.qld.gov.au/
probably we have uncovered an issue that squiz matrix didn't sync the jQuery version with swe